apps/ima/behaviours/save-image.js

```diff
@@ -97,3 +101,108 @@ module.exports = name => superclass => class extends superclass {
     return super.saveValues.apply(this, arguments);
   }
 };
+
+
+// 'use strict';
+
+// const _ = require('lodash');
+// const config = require('../../../config');
+// const Model = require('../models/image-upload');
+
+// const fileSizeNum = size => size.match(/\d+/g)[0];
+
+// module.exports = name => superclass => class extends superclass {
+//   process(req) {
+//     if (req.files && req.files[name]) {
+//       // set image name on values for filename extension validation
+//       // N:B validation controller gets values from
+//       // req.form.values and not on req.files
+//       req.form.values[name] = req.files[name].name;
+//       req.log('info', `Submission ID: ${req.sessionModel.get('submissionID')},
+//                        Processing image: ${req.form.values[name]}`);
+//     }
+//     super.process.apply(this, arguments);
+//   }
+
+//   locals(req, res, next) {
+//     const maxNum = fileSizeNum(config.upload.maxFileSize);
+//     const maxSize = config.upload.maxFileSize.match(/[a-zA-Z]+/g)[0].toUpperCase();
```

Comment:
Its probably best to remove this commented/duplicate code for tidiness. If you wanted to keep the slight differences as a reference for later maybe make a note of it elsewhere.